packages/shared/src/components/profile/UserShortInfoPlaceholder.tsx

```diff
@@ -19,7 +29,7 @@ export function UserShortInfoPlaceholder({
         .fill(0)
         .map((_, i) => (
           // eslint-disable-next-line react/no-array-index-key
-          <PlaceholderList key={i} />
+          <Placeholder key={i} />
```

Comment:
This was using the card list placeholder which didn't even match, so made a new matching one 🙏